tests/test_szp_meas_report/README.md

```diff
@@ -0,0 +1,135 @@
+<!-- 
+To compile puml use: (Assuming plantuml you are in the directory)
+plantuml.jar -tsvg README.md -o diagrams
+-->
+
+
+# Test on the SoundZone-protocol package
+
+This test will measure some key values that will be used to define the values used in the 
+firmware, and some measurements will be used in order to check that it is possible to
+stay within spec. 
+
+## Todo
+- [ ] Define file corruption
+
+## What measurements are interesting
+
+| Measurement | Description |
+|---|---|
+| Packet loss | How many packets are lost under transmission |
+| File corruptness | How much of the file has been corrupted under transmission |
```

Comment:
Hvad er forskellen på packet loss og file corruptness? For vil det ikke ses som det samme når det måles?

Hvis checksummen ikke passer vil det vel betegnes som en tabt pakke? Men det vil vel også svare til noget at pakke er corrupt